vyper/ast/nodes.py

```diff
@@ -1377,6 +1380,16 @@ class ImportFrom(_Import):
     __slots__ = ("level", "module")
 
 
+class ImplementsDecl(Stmt):
+    __slots__ = ("target", "annotation", "simple")
```

Comment:
do we allow non-simple targets?

Reply:
hm nope, will make the change and add comments.